config/app.php

```diff
@@ -112,6 +112,14 @@
 
     /*
     |--------------------------------------------------------------------------
+    | Upload link
+    |--------------------------------------------------------------------------
+    |
+    */
+    'upload' => 'backend/upload/',
```

Comment:
don't put the upload config in `config/app.php` please separate to another file, example `config/upload.php`